app/src/main/java/com/sdp13epfl2021/projmag/adapter/ProjectAdapter.kt

```diff
@@ -142,6 +142,7 @@ class ProjectAdapter(
         holder.commentButton.setOnClickListener {
             val context = holder.view.context
             val intent = Intent(context, CommentsActivity::class.java)
+            intent.putExtra("projectId", project.id)
```

Comment:
We should avoid string literal for extra name.
```suggestion
            intent.putExtra(MainActivity.projectIdString, project.id)
```